wire-ios-data-model/Source/Model/FeatureConfig/E2EId/Feature.E2EId.swift

```diff
@@ -46,13 +46,15 @@ public extension Feature {
 
         public struct Config: Codable, Equatable {
 
-            public let acmeDiscoveryUrl: String
+            public let acmeDiscoveryUrl: String?
```

Comment:
**nitpick(non-blocking):**
```suggestion
            public let acmeDiscoveryPath: String?
```

Reply:
You're right about the name, but `acmeDiscoveryUrl` is the key we get from BE. To parse the response, I need to provide the correct key name anyway.